commcare_connect/opportunity/tests/test_visit_import.py

```diff
@@ -290,17 +290,44 @@ def test_bulk_update_payments(opportunity: Opportunity):
     access_objects = []
     for mobile_user in mobile_user_seen:
         access_objects.append(OpportunityAccessFactory(opportunity=opportunity, user=mobile_user))
-    dataset = Dataset(headers=["Username", "Phone Number", "Name", "Payment Amount"])
+
+    dataset = Dataset(
+        headers=[
+            "Username",
+            "Phone Number",
+            "Name",
+            "Payment Accrued",
+            "Payment Completed",
+            "Payment Amount",
+            "Paid date",
+        ]
+    )
+
+    payment_date = "2025-01-15"
     for mobile_user in chain(mobile_user_seen, mobile_user_missing):
-        dataset.append((mobile_user.username, mobile_user.phone_number, mobile_user.name, 50))
+        dataset.append(
+            (
+                mobile_user.username,
+                mobile_user.phone_number,
+                mobile_user.name,
+                100,  # Payment Accrued
+                0,  # Payment Completed
+                50,  # Payment Amount
+                payment_date,
+            )
+        )
 
     payment_import_status = _bulk_update_payments(opportunity, dataset)
+
     assert payment_import_status.seen_users == {user.username for user in mobile_user_seen}
     assert payment_import_status.missing_users == {user.username for user in mobile_user_missing}
+
     assert Payment.objects.filter(opportunity_access__opportunity=opportunity).count() == 5
+
     for access in access_objects:
         payment = Payment.objects.get(opportunity_access=access)
         assert payment.amount == 50
+        assert payment.date_paid.strftime("%Y-%m-%d") == payment_date
```

Comment:
It would be nice to test the default value as well

Reply:
Done